helix-view/src/editor.rs

```diff
@@ -558,7 +558,7 @@ impl Default for WhitespaceCharacters {
 pub struct IndentGuidesConfig {
     pub render: bool,
     pub character: char,
-    pub skip_levels: u16,
+    pub skip_levels: usize,
```

Comment:
Unlikely that anyone will specify a `skip_levels` higher than `u8`. I'd store a `u8` here and cast it to `usize`  when calculating `let starting_indent`